service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/AbstractDocmosisData.java

```diff
@@ -5,8 +5,10 @@
 
 import java.util.Map;
 
-public abstract class AbstractDocmosisData {
+public class AbstractDocmosisData {
+
     public Map<String, Object> toMap(ObjectMapper mapper) {
-        return mapper.convertValue(this, new TypeReference<>() {});
+        return mapper.convertValue(this, new TypeReference<>() {
+        });
```

Comment:
Seems like unnecessary change

Reply:
ahh damn, reformat code does this for me by default, shall revert. Good spot